lldb/source/API/SBCommandInterpreter.cpp

```diff
@@ -743,3 +743,15 @@ void SBCommand::SetFlags(uint32_t flags) {
   if (IsValid())
     m_opaque_sp->GetFlags().Set(flags);
 }
+
+void SBCommandInterpreter::SetPrintCallback(
+    lldb::SBCommandPrintCallback callback, void *baton) {
+  LLDB_INSTRUMENT_VA(this, callback, baton);
+
+  if (m_opaque_ptr)
+    return m_opaque_ptr->SetPrintCallback(
```

Comment:
this isn't actually returning anything

```suggestion
    m_opaque_ptr->SetPrintCallback(
```